pkg/common/kafka/testing/mocks.go

```diff
@@ -1,5 +1,5 @@
 /*
-Copyright 2020 The Knative Authors
+Copyright 2021 The Knative Authors
 
```

Comment:
Did moving the file cause this?  I don't think we are supposed to roll the original copyright dates forward as we edit files?

Reply:
I moved this file several times; it wouldn't surprise me, but I will replace it.